src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -8,23 +8,43 @@ import java.nio.charset.Charset
 
 class Lambda {
 
-  def process(inputStream: InputStream, outputStream: OutputStream) = {
-    val apiUser = createUser(lambdaConfig.consignmentApiUser)
+  def process(inputStream: InputStream, outputStream: OutputStream): Unit = {
+    createUsers(lambdaConfig.databaseName)
+
+    outputStream.write("Users created successfully".getBytes(Charset.defaultCharset()))
+  }
+
+  def createUsers(databaseName: String): Boolean = {
+    databaseName match {
+      case "consignmentapi" => createConsignmentApiUsers
+      case "keycloak" => createKeycloakUsers
+    }
+  }
+
+  def createKeycloakUsers: Boolean = {
+    val user = sqls.createUnsafely(lambdaConfig.keycloakUser)
+    val password = sqls.createUnsafely(lambdaConfig.keycloakPassword)
+    sql"CREATE USER $user WITH PASSWORD '$password'".execute().apply()
```

Comment:
Would it make sense to abstract out the  `CREATE USER`, `GRANT CONNECT ON DATABASE` and `GRANT USAGE ON SCHEMA public TO` sql  statements to a separate function as they are essentially the same in the `createKeycloakUsers` and `createConsignmentApiUser` functions? Passing in the relevant params to the abstracted function?

Reply:
I've moved `GRANT CONNECT` and `GRANT USAGE` to a new method. The `CREATE USER` calls are slightly different as keycloak is getting a password and the consignment ones aren't so I've left those.